pkg/config.go

```diff
@@ -22,24 +22,32 @@ func (c *Client) getConfigMap(namespace, name string) (configMap *ConfigMap, err
 	return
 }
 
+// GetSystemConfig loads various system configurations and bundles them into a map.
+// The configuration is cached once it is loaded, and that cached value is used from here on out.
 func (c *Client) GetSystemConfig() (config map[string]string, err error) {
-	namespace := "onepanel"
-	configMap, err := c.getConfigMap(namespace, "onepanel")
-	if err != nil {
-		return
-	}
-	config = configMap.Data
+	if c.systemConfig == nil {
```

Comment:
We need to figure out a better way to cache this for the entire application and not just at request level.